src/container/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(hash)
```

Comment:
This is one example of places where I felt that we might be able to organize things better.  But it's probably not worth changing now.

Reply:
I'll look into it later :)